cicd/pipelines/working-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -11,10 +11,10 @@ You can use the *Developer* perspective of the {product-title} web console to cr
 
 In the *Developer* perspective:
 
-* Use the *Add* -> *Pipeline* -> *Pipeline Builder* option to create customized pipelines for your application.
+* Use the *Add* -> *Pipeline* -> *Pipeline builder* option to create customized pipelines for your application.
```

Comment:
@masauso-lungu How is it appearing in the UI? Please check and do the needful accordingly.